sql_generators/stable_views/__init__.py

```diff
@@ -109,7 +103,13 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, schema: SchemaF
     """If a view.sql does not already exist, write one to the target directory."""
     # add imports here to run in multiple processes via pathos
     from bigquery_etl.format_sql.formatter import reformat
+    from bigquery_etl.schema import Schema
```

Comment:
Were are using `pathos` for running the generation in parallel. It cannot find global imports for some reason (I'll investigate further), so moving imports into methods.